addon_development.md

```diff
@@ -0,0 +1,32 @@
+# Addon Development Guidelines
+
+We welcome any addon contributions into the open-cluster-management community!
+
+This document outline a structure for developing an addon in the Open Cluster Management project.
+This ensures a clear clarification of goals/non-goals of addons,
+ensures the addon maintainers working with stable commitment, ensures a rapid response
+to potential required fixes (e.g. critical security problems) and (most importantly)
+it ensures that contributors and users receive quick feedback on their issues and contributions.
+
+## Scope of an addon
+
+- The addon must provide management capability over multiple kubernetes clusters.
+- It can be a brand new project or an integration with existing project
+
+## Rules for New Addons
+
+* The name of an addon code repo in github.com/open-cluster-management-io must have `-addon` as the suffix
```

Comment:
I'd also encourage that whatever the add on name is when installing it should match the repo name other than the `-addon` suffix.

Reply:
That sounds reasonable but may not be a requirement?